ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml

```diff
@@ -107,16 +110,102 @@ StackView {
         id: newAirdropView
 
         SettingsPage {
+            id: newAirdropPage
             title: qsTr("New airdrop")
 
             contentItem: EditAirdropView {
                 id: view
 
                 padding: 0
 
+                Loader {
```

Comment:
Wrapping here into `Loader` doesn't bring any benefit because it's already in `Component` which is already lazily instantiated on demand.